skymp5-server/cpp/server_guest_lib/MpChangeForms.h

```diff
@@ -71,6 +71,9 @@ class MpChangeForm : public MpChangeFormREFR
 
   static nlohmann::json ToJson(const MpChangeForm& changeForm);
   static MpChangeForm JsonToChangeForm(simdjson::dom::element& element);
+
+public:
+  std::optional<int> lockLevel = 0;
```

Comment:
Hmm, does it need to be attached to any form? And be public? And probably by default it should be `nullopt` instead of `0`?